src/database/to-dos.js

```diff
@@ -1,42 +1,55 @@
-const {getDatabase} = require('./mongo');
-const {ObjectID} = require('mongodb');
+const { getDatabase } = require('./mongo');
+const { ObjectID } = require('mongodb');
+const { paginate } = require('./../helper');
```

Comment:
if it is a level down, you can simply: `../helper` instead of `./../helper`

Reply:
Okay